handlers/rules/expressions/isSamlLoginRequest.cfc

```diff
@@ -0,0 +1,22 @@
+/**
+ * @expressionCategory browser
+ * @expressionContexts webrequest
+**/
+component {
+	/**
+	 * @serviceProviders.fieldType object
+	 * @serviceProviders.object    saml2_consumer
+	 * @serviceProviders.multiple  true
+	 */
+	private boolean function evaluateExpression( event, rc, prc, boolean _is=true, string serviceProviders="" ){
+		if ( !isStruct( rc.samlRequest ?: "" ) ) {
```

Comment:
This looks fantastic. Great work making this concise. However, it is missing taking the `_is` into account. I suggest refactoring to something along the lines of:

```cfc
var isSamlLoginRequest = false;
if ( IsStruct( rc.samlRequest ?: "" ) ) {
    if ( !Len( arguments.serviceProviders ) || ListFindNoCase( ... ) ) {
        isSamlLoginRequest = true;
    }
}
return isSamlLoginRequest == arguments._is;
```